lib/va_profile/models/associated_person.rb

```diff
@@ -1,58 +1,35 @@
 # frozen_string_literal: true
 
-require_relative 'base'
-require 'common/models/attribute_types/iso8601_time'
+require 'common/models/base'
+require 'common/models/attribute_types/titlecase_string'
 
 module VAProfile
   module Models
     class AssociatedPerson < Base
-      PRIMARY_NEXT_OF_KIN = 'Primary Next of Kin'
-      OTHER_NEXT_OF_KIN = 'Other Next of Kin'
       EMERGENCY_CONTACT = 'Emergency Contact'
       OTHER_EMERGENCY_CONTACT = 'Other emergency contact'
+      PRIMARY_NEXT_OF_KIN = 'Primary Next of Kin'
+      OTHER_NEXT_OF_KIN = 'Other Next of Kin'
 
       CONTACT_TYPES = [
-        PRIMARY_NEXT_OF_KIN,
-        OTHER_NEXT_OF_KIN,
         EMERGENCY_CONTACT,
-        OTHER_EMERGENCY_CONTACT
+        OTHER_EMERGENCY_CONTACT,
+        PRIMARY_NEXT_OF_KIN,
+        OTHER_NEXT_OF_KIN
       ].freeze
 
       attribute :contact_type, String
-      attribute :given_name, String
-      attribute :middle_name, String
-      attribute :family_name, String
-      attribute :relationship, String
+      attribute :given_name, Common::TitlecaseString
+      attribute :middle_name, Common::TitlecaseString
+      attribute :family_name, Common::TitlecaseString
```

Comment:
Some people are going to encounter issues with last names like O'Shea, McDonald, etc 😞 Unfortunately I don't know a way around with it all caps coming back from the service.